react-components/src/apps/admin/AdminAppDetails.js

```diff
@@ -312,6 +314,6 @@ AdminAppDetailsDialog.propTypes = {
 
 export default withFormik({
     enableReinitialize: true,
-    mapPropsToValues: ({ app }) => ({ ...app }),
+    mapPropsToValues: ({ details }) => ({ ...details }),
```

Comment:
I think this will break a few things in the AdminAppDetails view.  `details` doesn't have the `beta` key, for example, and the form is looking for that.  Also, in the `handleSubmit` method I added a lot of logic checking the `props` (initial) values vs the form's (updated) `values` to try and minimize the number of calls we were making when submitting the form.

We can probably do this:
```
mapPropsToValues: ({ app, details }) => ({ ...app, ...details }),
```
and then update the `handleSubmit` method to check the right `props` (now it will have to check `props.details` in some spots, instead of `props.app`).

Reply:
Good catch...i had my doubts when modifying this. Will add a fix for this.